src/main/java/com/example/automator/helper/DataInput.java

```diff
@@ -130,4 +134,66 @@ public void setPercentageNegativeWoM(String percentageNegativeWoM) {
     public void setBaseAdoptionProbability(String baseAdoptionProbability) {
         this.baseAdoptionProbability = baseAdoptionProbability;
     }
+
+    public String isDataInputValid(DataInput dataInput) { // avgIntraMentionPercentage, percentageNegativeWoM, baseAdoptionProbability, nrDefaultFriendsInterVillage, avgIntraVillageInteractionFrequency, avgInterVillageInteractionFrequency, avgChiefFarmerMeetingFrequency
+
+        // no empty strings
+        try {
+        Assert.hasText(dataInput.getNrDefaultFriendsInterVillage(), "nrDefaultFriendsInterVillage is empty");
```

Comment:
Ich würde die Message jeweils so formulieren, dass noch klarer ist, wo der Fehler liegt.
Zudem würde ich nennen, was nicht sein darf, anstatt, was gerade der Fall ist. Hier z.B. statt "is empty", "must not be empty".